week2/assignment/css/style.css

```
@@ -0,0 +1,161 @@
+body {
+    background-color: #d5e9ff;
+}
+
+header {
+    background-color: #71b6ff;
+    color: white;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    align-items: center;
+    padding: 1.5rem;
+
+    & i {
+        font-size: 1.25rem;
+        justify-self: start;
+    }
+
+    & h1 {
+        font-size: 1.5rem;
+        font-weight: bold;
+        justify-self: center;
+    }
+
+    & h3 {
+        font-size: 1rem;
+        justify-self: end;
+    }
+}
+
+main {
+    width: 65%;
+    margin: auto;
+}
+
+.filter-container {
+    background-color: #c6e1ff;
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    gap: 0.8rem;
+    padding: 1.5rem;
+    margin-top: 2rem;
+}
+
+.filter {
+    display: flex;
+
+    label {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        background-color: #71b6ff;
+        width: 8.5rem;
+        height: 4.3rem;
+        font-size: 1rem;
+        font-weight: bold;
```

p2: `font-weight: bold`보다는 정확한 수치를 나타내주는 것이 일관성 유지, 정확한 표현 측면에서 더 좋은거 같아요!
```suggestion
        font-weight: 700;
```
추가적으로, bold는 700으로 매핑되고, normal은 400으로 매핑된다고 하니 참고하세요!

---

습관처럼 bold라고 사용했었는데 수치로 나타내주는게 더 낫나보네요 ! 반영했습니당 🫠